src/main/java/pl/wedding/management/weddinghall/WeddingHall.java

```diff
@@ -0,0 +1,27 @@
+package pl.wedding.management.weddinghall;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+public class WeddingHallController {
+
+    @Autowired
```

Comment:
Can we do it less "auto-magic-spring-way" that makes writing test hard, but more Java-way: constructor dependency injection.
Having constructor dependency injection makes:
- Easier to test such class, as you can pass your implementation/mocks/spy
- Easier to resign from spring and move to some other libraries
- Easier to control what bean and in what number are discovered by spring by explicit creation of @Bean

Reply:
Ok, i changed it. Check this out and give feedback.